packages/block-library/src/post-comments-form/edit.js

```diff
@@ -67,8 +70,38 @@ export default function PostCommentsFormEdit( {
 					</Warning>
 				) }
 
-				{ ( 'open' === commentStatus || isInSiteEditor ) &&
-					__( 'Post Comments Form' ) }
+				{ ( 'open' === commentStatus || isInSiteEditor ) && (
+					<div className="wp-block-post-comments-form">
```

Comment:
I think `blockProps` (used on [L52](https://github.com/WordPress/gutenberg/pull/40368/files#diff-abf88ae09f89f6af7894b1f8480350eb820df74aff7e2b61490e93dd48c4722fR52)) already contains the block classname.
![Bildschirmfoto 2022-04-21 um 20 39 12](https://user-images.githubusercontent.com/96308/164530937-4266d191-5dec-4788-b1b4-52070eaebc44.png)

We should probably remove it here:

```suggestion
					<div>
```

Reply:
done in 4e1c59b9d45a127d39c20e77c2a09d700eb08d8d